http/1.5/modules/ROOT/pages/http-post-form-task.adoc

```diff
@@ -1,111 +1,74 @@
-+
-[source,text,linenums]
-----
-Sorry #[payload.'firstname'], you're too young to register.
-----
+.Post a Form flow
+image::http-example-post-form-flow.png[Post a Form flow]
 
-+
-Note that this expression refers to one of the form parameters included in the request: `firstname`. After the connector receives the parameter, it exists in the Mule Message payload as an item in a map
-. On the Choice router, configure the *When* condition for that path as follows:
-+
-[source,text,linenums]
-----
-#[server.dateTime.year-18 < payload.'yearborn']
-----
+== Create the Mule Application
 
-+
-Once again, this expression refers to one of the incoming form parameters, `yearborn`.
-. Complete the flow by adding another Set Payload element in the *Default* path of the Choice router, with a payload value of:
-+
+To create the Mule flow:
 
-[source,text,linenums]
-----
-Registration has been carried out successfully! Welcome #[payload.'firstname'] #[payload.'lastname']!
-----
-
-.Post Form Flow.
-image::http-example-post-form-flow.png[HTTP Post form flow]
-
-Once you create the flow, you can <<post-form,post a form by sending HTTP requests>>.
-
-== Create an App with HTTP Listener Connector in XML Editor
-
-. Create an `<http:listener>` global configuration and set *host* to `localhost` and *port* to `8081`.
-. Create a flow with an `<http:listener>` connector. Reference your connector to the global element you just created, set *path* to `submitform`.
+. In the *Mule Palette* view, select the HTTP *Listener* source and drag it on to the canvas. +
+The source initiates the flow by listening for incoming HTTP message attributes.
+. Set the *Path* field to `submitform`.
+. Click the plus sign (*+*) next to the *Connector configuration* field to configure a global element that can be used by all instances of the HTTP *Listener* in the app.
+. On the *General* tab, configure the following fields:
 +
-
-[source,xml,linenums]
-----
-<http:listener-config name="HTTP_Listener_Configuration" host="localhost" port="8081" doc:name="HTTP Listener Configuration"/>
-
-    <flow name="RegisterUser">
-        <http:listener config-ref="HTTP_Listener_Configuration" path="submitform" doc:name="HTTP"/>
-
-    </flow>
-----
-
-. In the flow, after the HTTP Connector add a `choice` router with two alternative paths, in which the first path executes when the following MEL expression is true:
+* *Host*: `localhost`
+* *Port*: `8081`
 +
+[start=5]
+. Click *OK*.
+. Drag a *Choice* router component to the right of the HTTP *Listener* source. +
+The *Choice* router adds conditional processing to the flow.
+. Drag a *Set Payload* component into the *When* condition box of the *Choice* router.
+. Set the *Value* field to `Sorry #[payload.'firstname'], you're too young to register.` +
+Note that this expression refers to one of the form parameters included in the request: `firstname`. After the connector receives the parameter, it exists in the Mule Message payload as an item in a map.
```

Comment:
For the following sentence:
After the connector receives the parameter, it exists in the Mule Message payload as an item in a map.

Could it be said the following for the end of the sentence? ...payload as an item within a map.
Replaced 'in' with 'within'.